future/src/ct/ct_misc_utils.cc

```diff
@@ -760,6 +728,17 @@ char* CtRgbUtil::set_rgb24str_from_str_any(const char* rgbStrAny, char* rgb24Str
     return rgb24StrOut;
 }
 
+std::vector<std::string> CtMiscUtil::split(const std::string& str, std::string_view sep) {
```

Comment:
there is `str::split`, why one more?

Reply:
Because I didn't know about `str::split` is why xD